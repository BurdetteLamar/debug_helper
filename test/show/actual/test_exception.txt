---
Exception (message='test_exception'):
  message: Boo!
  cause: 
