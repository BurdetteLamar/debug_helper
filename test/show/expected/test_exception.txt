---
Exception (message='test_exception'):
  Exception#message: Boo!
  Exception#cause: 
